app/services/problem_check/channel_errors.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ProblemCheck::ChannelErrors < ProblemCheck
```

Comment:
This gets added into the `ProblemCheck` registry when inherited. No need to explicitly add it.